components/server/src/ome/security/basic/CurrentDetails.java

```diff
@@ -302,7 +302,7 @@ public Event newEvent(Session session, EventType type,
         // Proxied if necessary
         e.setExperimenter(c.getOwner());
         e.setExperimenterGroup(c.getGroup());
-        e.setSession(session);
+        e.setSession(new Session(session.getId(), false));
```

Comment:
Do you happen to still recall why is it necessary to unload the session at this point?